app/controllers/editions_controller.rb

```diff
@@ -71,6 +93,14 @@ def setup_view_paths
     setup_view_paths_for(resource)
   end
 
+  def update_action_is_publish?
```

Comment:
Since we don't have a way to publish yet, do we need this method?

If we do, I think this method can be made `private` (which it is in the legacy controller).

Reply:
I believe we do not, nor do we need `UpdateWorker.perform_async(resource.id.to_s, update_action_is_publish?)` above that, so removing both.